packages/interfaces/src/interval.interface.ts

```diff
@@ -0,0 +1,23 @@
+export interface Interval<T = number | string> {
```

Comment:
If we add a docbloc comment here, will that show up in the docs as the description for the whole interface? Asking because it seems to me like this might need at least _some_ description for people to know what it's for.

```suggestion
/**
 * <description here>
 */
export interface Interval<T = number | string> {
```